core/chain-configs/src/genesis_config.rs

```diff
@@ -13,15 +13,17 @@ use num_rational::Rational;
 use serde::{Deserialize, Serialize};
 use smart_default::SmartDefault;
 
-use near_primitives::hash::CryptoHash;
-use near_primitives::serialize::{u128_dec_format, u128_dec_format_compatible};
-use near_primitives::state_record::StateRecord;
-use near_primitives::types::{
-    AccountId, AccountInfo, Balance, BlockHeight, BlockHeightDelta, EpochHeight, Gas, NumBlocks,
-    NumSeats,
+use near_primitives::{
+    hash::CryptoHash,
+    runtime::config::RuntimeConfig,
+    serialize::{u128_dec_format, u128_dec_format_compatible},
+    state_record::StateRecord,
+    types::{
+        AccountId, AccountInfo, Balance, BlockHeight, BlockHeightDelta, EpochHeight, Gas,
+        NumBlocks, NumSeats,
+    },
+    version::ProtocolVersion,
```

Comment:
I vote to revert styling changes back unless we commit to a certain style.

Reply:
Yeah good point; this was just hand crafted a while ago, but I'll revert!

Reply:
fwiw, I have a "more than rustfmt" style guide here: https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md